### ISSUE TITLE
[$30][MSFT-231] [Challenge Feed][NVD] Prizes are incorrectly read-out by the Screen reader

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2889]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges
2. Hover over the purse and then hover over the prizes

**Expected Result**
Should read-out the prizes correctly

**Actual Result**
Prizes are incorrectly read-out by the Screen reader

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F67

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096023 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096023 has been updated - it has been assigned to daga_sumit.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by sumitdaga** [comment]
@veshu  
i cannot reproduce this issue, can you give an example when it does not work!

**Comment by veshu** [comment]
@sumitdaga Are you using this tool https://www.nvaccess.org/?

**Comment by sumitdaga** [comment]
@veshu no i am using the [narrator](https://support.microsoft.com/en-in/help/22798/windows-10-complete-guide-to-narrator) app that comes with windows 10

**Comment by veshu** [comment]
@sumitdaga This was raised using [NVD] tool. Some member has used title to define the tool they have used. Please check with NVD once, it must be replicable. Else we can ask QA reviewer to confirm.

Have you checked the video in attachment? It is reading improperly.

**Comment by sumitdaga** [comment]
@veshu  i think i don't have access to the attachment video!  
but i used the NVD tool and it seems to read the correct amount

**Comment by veshu** [comment]
@sumitdaga https://apps.topcoder.com/forums/?module=Thread&threadID=940852&start=0

There are credentials shared for one drive.  

The issue is it is not reading correctly. like it should be able read as "First prize is $1000, second prize 500" on hover but it said "one $1000, 2 500". Please check the video.

Regarding assignment yes you can pick another. I will get confirm with QA member till. Thanks!

**Comment by veshu** [comment]
@drasticdpk @tosha5252 Can you please check if above issue is replicable or not.

**Comment by PrakashDurlabhji** [comment]
@veshu I am able to replicate hence trying to solve it

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096023 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096023 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by rashmi73** [comment]
@veshu @codeMinter ready https://github.com/topcoder-platform/community-app/pull/2980

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2889 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2889 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2889 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2889 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2889 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097071 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097071 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by rashmi73** [comment]
@veshu yes updating, and i will paste verification video for same

**Comment by rashmi73** [comment]
@veshu suddenly it has stopped working, i remember it was working well. watching it

**Comment by veshu** [comment]
@rashmi73 Any update on this thanks!

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097071 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by drasticdpk** [comment]
Verified on 
Operating System: Windows
Browser: Chrome

**Comment by lakshmiathreya** [comment]
Prizes are ready correctly on mouse hover. However, Prizes are skipped when tabbing from Skill list to # of participants. Hence setting to Fail.

**Comment by gets0ul** [comment]
@lakshmiathreya 
That is out of scope of this issue requirement.
The expected result is to read out the prize in natural language on hovering.

**Comment by lakshmiathreya** [comment]
@gets0ul - since 'Prizes being skipped' is addressed by #2828, closing this Issue.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097071<br/><br/>```This is an automated message for crazyk via Topcoder X```
